app/models/backup.rb

```diff
@@ -24,6 +24,27 @@ def self.[](filename)
     end
   end
 
+  def self.get_file_from_filename(filename)
```

Comment:
Based on https://github.com/discourse/discourse/pull/4718#issuecomment-282888181, I think the following methods are not required anymore?